Library/ViewModels/PledgePaymentMethodsViewModel.swift

```diff
@@ -16,12 +16,20 @@ public typealias PledgePaymentMethodsValue = (
   project: Project,
   reward: Reward,
   context: PledgeViewContext,
-  refTag: RefTag?
+  refTag: RefTag?,
+  pledgeTotal: Double,
```

Comment:
Passing the `pledgeTotal` in this way is a bit ugly, since it means I'm setting it to an empty value for non-late-pledge usages of this screen. I tried to document it in the code with `assert`s.